manual/linker.wiki

```diff
@@ -2,23 +2,19 @@
 The Js_of_ocaml compiler accepts JavaScript files provided on the command-line.
 The main purpose is to provide (external) primitives needed by the bytecode program.
 
-Most of the primitives from the standard library are already implemented and loaded by default (located in "runtime.js").
+Most of the primitives from the standard library are already implemented and loaded by default.
 
-Additionally, some other primitives are installed but not loaded by default:
- * "+toplevel.js" : when compiling toplevel
+Additionally, some other primitives are available but not loaded by default:
+ * "+toplevel.js" and "+dynlink.js" when compiling toplevel and/or using dynlink.
```

Comment:
I think we should expose (ocaml empty) libraries to bring these in rather than use this special syntax. See my comment below.

Reply:
#1324 adds `+dynlink.js` and `+toplevel.js` whenever  `--toplevel` or `--dynlink` is used. More cleanup is required, the distinction between `--toplevel` and `--dynlink` should be documented or removed.

Reply:
If there's no distinction then indeed it may be worth deprecating `--toplevel` and instruct people to use `--dynlink` when they want to compile a toplevel or use the `Dynlink` library (it's not entirely unnatural IIRC ocamlnat uses `Dynlink`).